### ISSUE TITLE
libjpeg62-turbo instead of libjpeg8

### ISSUE THREAD

**Issue by lucaarcteam** [archeos/cmvspmvs-archeos#3]
It seems that libjpeg8 has been replaced by libjpeg62-turbo in Debian Jessie. Should we update the code for this?

**Comment by ORNis** [comment]
maybe we should also migrate to the last version of cmvs-pmvs : the HEAD of https://github.com/pmoulon/CMVS-PMVS.git

**Comment by lucaarcteam** [comment]
OK, I am downloading it right now. I'll test it and send a report.

**Comment by lucaarcteam** [comment]
I just compiled and tested the software of Pierre. Everythign is working perfectly

**Comment by ORNis** [comment]
Yes, as usual with Pierre. In fact last patches didn't came from him but they greatly simplifies the compilation on modern OSes as they drops some kind of exotic dependencies.
(PS : sorry for closing the issue)
